app/src/main/java/com/kickstarter/libs/AnalyticEvents.kt

```diff
@@ -271,6 +272,19 @@ class AnalyticEvents(trackingClients: List<TrackingClientType?>) {
         client.track("Application Login or Signup", properties)
     }
 
+    fun trackLoginOrSignUpCtaClicked(type: String, page: String) {
+        val props: HashMap<String, Any> = hashMapOf(CONTEXT_CTA.contextName to LOGIN_OR_SIGN_UP.contextName)
+        props[CONTEXT_PAGE.contextName] = page
+        type.let { props[CONTEXT_TYPE.contextName] = type }
```

Comment:
why using .let in a non optional ?

Reply:
The method is called from a java class. I had to check for null

Reply:
that's ok, then declare on the kotlin side he parameter as optional, and unwrap it 👍  ` fun trackLoginOrSignUpCtaClicked(type: String?, page: String)`

Reply:
thanks